packages/cli/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@quality-metrics/cli",
+  "version": "0.0.1",
+  "dependencies": {
+    "bundle-require": "^4.0.1"
```

Comment:
Though this was added automagically by nx

Reply:
As mentioned above, Nx has a lint rule for this now. It seems like they were generating it before (e.g. there's [a deprecated `generatePackageJson` executor option](https://github.com/nrwl/nx/blob/master/packages/esbuild/src/executors/esbuild/esbuild.impl.ts#L77C5-L80)), but I guess they had a reason for moving away from that.